docs/developers/deploy-on-polygon.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Deploying the LSP on Polygon
+sidebar_label: Deploying the LSP on Polygon
+---
+UMA's infrastructure is also available on Polygon for developers to make use of as a scaling solution. The following will explain the details on how to deploy on Polygon. 
+
+Deploying to Polygon follows a near-identical launch process as to launching any UMA contract on Ethereum. Using one of our launch repo's and amending the destination network, you can quickly deploy your contracts on Polygon.
+
+## How UMA works on Polygon
+
+UMA makes use of an arbitrary message bridge that allows for two-way messages to pass between the Polygon network and Ethereum mainet. The message bridge makes use of the Polygon Arbitrary Message system, meaning it can be as trusted as the Polygon network itself. 
+
+UMA's Optimistic Oracle is used as the arbiter of price requests locally on the Polygon network. A request will ask the locally deployed Optimitic Oracle. If the proposal goes undisputed, then the result is deemed the accepted outcome. 
```

Comment:
What does "locally deployed OO" mean? Locally usually means in a test environment on a developer's computer.

Reply:
changed